### ISSUE TITLE
Simple validation fails :(

### ISSUE THREAD

**Issue by tamilvjm** [syrusakbary/validate_email#104]
Hi,

I have just validated with simple invalid email and it is showing as valid one. Could you explain and fix?

`>>> from validate_email import validate_email
>>> is_valid = validate_email('example@example.com')
>>> is_valid
True
>>> is_valid = validate_email('example@example')
>>> is_valid
True
>>>
`

**Comment by JSquar** [comment]
`validate_email('example@example')` without setting `verify` attribute to `True` only evaluates if the mail conforms with the general mail regex.

**Comment by ps91** [comment]
> 
> 
> `validate_email('example@example')` without setting `verify` attribute to `True` only evaluates if the mail conforms with the general mail regex.

Shouldn't the general mail regex assume that after the "@" there's at least "letters" "dot" "letters" once?

**Comment by b4stien** [comment]
> Shouldn't the general mail regex assume that after the "@" there's at least "letters" "dot" "letters" once?

No, it can not assume that. You can be delivering mail to "localhost", or you can have a special DNS that does not have a tld (the part after the dot).
